include/swift/AST/Expr.h

```diff
@@ -154,12 +154,12 @@ class alignas(8) Expr : public ASTAllocated<Expr> {
     Implicit : 1
   );
 
-  SWIFT_INLINE_BITFIELD_FULL(CollectionExpr, Expr, 64-NumExprBits,
+  SWIFT_INLINE_BITFIELD_FULL(CollectionExpr, Expr, 64-NumberOfExprBits,
```

Comment:
Could we perhaps namespace this as `swift::Expr::NumExprBits` or some such?

Reply:
I don’t think we can extrapolate this approach to the macros, though. The subclass bitfield macros have code that should expand to the parent class’s `Num*Bits`.

Reply:
I'm not sure I follow. If the member is a `public: static const constexpr size_t NumExprBits = ...` that should be accessible by name for the sub types.

Reply:
Right, but it won’t prevent the preprocessor from expanding `Num##U##Bits` to whatever clang defined, will it?